reports/in-app-guidance-shepherd/README.md

```diff
@@ -0,0 +1,519 @@
+
+So something like this:
+
+---
+
+    <Pointing to the bar graph element>
+
+    **Product Ratings**
+
+    This graph displays product ratings as horizontal bars. This allows you to quickly parse how different products stack up against each other.
+
+    This graph is also **responsive to data changes on this page**. If you edit a product's rating in the table below, this graph will automatically update to reflect the new rating.
+
+---
+
+    <Pointing to donut chart element>
+
+    **Next step title**
+
+    Body text here…
+
+This helps reviewers parse the document as individual steps instead of one big page of prose. **Which, by the way, have other people review and QA your work!** When script writing, it's easy to go on autopilot and not consider how each user may read your script or interact with the page, so get feedback early and often. Google Docs makes quick collaboration very easy, so try to have as many reviewers as you can.
+
+After that work is done, and you have ✨ the script ✨ prepared, let's get down to business on prepping our Skuid page for our tour.
+
+## The Skuid work
+
+### Add unique IDs to elements
+
+Shepherd works by targeting DOM elements and using [the Popper.js library](https://popper.js.org/) to render step dialog boxes. While CSS class selectors can work, the best way to reliably target specific DOM elements is through **unique IDs**. This is great because, as a Skuid developer, you can apply unique IDs in _a lot_ of places.
+
+Let's apply these unique IDs to the elements we noted in our outline above (Not sure how? Check the component doc links below for the **Unique ID** property):
+
+- [Chart component 1](https://docs.skuid.com/latest/v2/en/skuid/components/ink/visualizations/chart/#advanced-tab): `products-by-rating`
+- [Chart component 2](https://docs.skuid.com/latest/v2/en/skuid/components/ink/visualizations/chart/#advanced-tab): `products-by-supplier`
+- [Table component](https://docs.skuid.com/latest/v2/en/skuid/components/ink/table/#display-tab): `product-list`
+- [Chart drilldown modal](https://docs.skuid.com/latest/v2/en/skuid/components/ink/modal/#advanced-tab)`: supplier-drilldown`
+
+We'll use these unique IDs in the Shepherd script code below.
+
+### Add Shepherd to the Skuid page
+
+To utilize the Shepherd library, we'll need to make sure it's in the Skuid page. This is done via the JavaScript resources tab.
+
+You could put the whole library inline, but I'm just going to use a CDN external resource.
+
+1. Add [a new JavaScript resource](https://docs.skuid.com/latest/v2/en/skuid/javascript/resource-types/).
+2. Set the following values:
+    1. **Resource Type**: URL
+    2. **Resource Name**: Shepherd
+    3. **Resource URL**: `https://cdn.jsdelivr.net/npm/shepherd.js@8.1.0/dist/js/shepherd.js`
+
+### Add some CSS rules for Shepherd's elements
```

Comment:
Can we poll the authorities - is there some better way to add custom vendor specific CSS to a page than the "text component advanced tab".   Ugh.  I feel so nasty,  even the pretty girls don't make it feel good.   Though its a nice meme.

Reply:
I'll ask the devs one more time, but I think we may be stuck getting in the Text component mobile with Regina George.